packages/documentation/.vitepress/theme/style.css

```
@@ -1,13 +1,120 @@
+  --vp-button-brand-hover-bg: var(--vp-c-brand-2);
+  --vp-button-brand-hover-border: var(--vp-c-brand-2);
+  --vp-button-brand-hover-text: #fffaf0;
+  --vp-button-brand-active-bg: var(--vp-c-brand-2);
+
+  /* Code */
+  --vp-code-color: #8a4a12;
+  --vp-code-bg: rgba(184, 101, 26, 0.08);
+
+  /* Typography */
+  --vp-font-family-base:
+    "Inter", ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto,
```

Stylelint (17.9.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)